src/res_cli.ml

```diff
@@ -243,8 +251,12 @@ module CliArgProcessor = struct
           else exit 1
         end
         else
+          let parsetree = match ppx with
```

Comment:
Can we keep this a string and match on the string for now? Does this extra conversion give us something in the end?